src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+.card {
+  width: $card-width;
+  height: $card-height;
+  background: $card-background-color;
+  border-radius: $card-radius;
+  border: 1px solid $card-grey-elements;
+
+&__bottom {
+  padding-inline: $card-bottom-padding-inline;
+}
+
+&__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  padding: 32px 19px 40px;
+}
+
+&__id {
+  margin: 0;
+  padding-bottom: 16px;
+
+  color: $card-secondary-accent;
+  font-family: inherit;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+&__title {
+  margin: 0;
+  padding-bottom: 4px;
+
+  color: $card-main-accent;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+&__review {
+  display: flex;
+  justify-content: space-between;
+  align-self: center;
+  padding-bottom: 24px;
+
+  &-count {
+    margin: 0;
+    font-family: inherit;
```

Comment:
it is inherited by default - no need to explicitly do it every time (only in case it's overwritten by browser default styles - you may check it in devTools in this case):
![image](https://github.com/mate-academy/layout_product-cards/assets/86923089/8cdcfbc4-c3d8-4076-8c3a-f54ba3460a5d)

[Article](https://www.w3schools.com/cssref/pr_font_font-family.php)